sno-plus/README.md

```diff
@@ -0,0 +1,203 @@
+# **Running sno with extra workers**
+
+This document provides information on how to deploy a SNO and add extra workers to it, and then study behaviour when loosing some of the workers or the initial node
+
+Note: appendix contains kcli commands used to speed up the workflow
+
+## **Environment**
+
+* We run all the deployments using libvirt vms, although the procedure would be the same for baremetal nodes.
+
+## **Initial deployment**
+
+1. Deploying SNO
+
+We leverage bootstrap in place procedure to generate a valid ignition file that is then embedded using coreos-installer
+
+We start with the following install config where:
+
+- we specify platform as None
+- we indicate in the bootstrap in place the disk to use for installation
+- we set network type to OVN
+
+```
+apiVersion: v1
+baseDomain: karmalabs.com
+compute:
+- name: worker
+  replicas: 0
+controlPlane:
+  name: master
+  replicas: 1
+metadata:
+  name: mycluster
+networking:
+  networkType: OVNKubernetes
+  machineNetwork:
+  - cidr: 10.0.0.0/16
+  clusterNetwork:
+  - cidr: 10.132.0.0/14
+    hostPrefix: 23
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  none: {}
+BootstrapInPlace:
+  InstallationDisk: /dev/vda
+pullSecret: 'XXX'
+sshKey: |
+  ssh-rsa ZZZ
+```
+
+With this install config, we then generate an ignition to use for SNO
+
+```
+mkdir ocp
+cp install-config.yaml ocp
+openshift-install --dir=ocp create single-node-ignition-config
+```
+
+Provided this content is stored in /root/config.ign, we inject it from a rhcos live iso using the following invocation
+
+```
+coreos-installer install --ignition=/root/config.ign /dev/vda
+```
+
```

Comment:
wouldn't be better to download the iso, embed the ignition file in the iso and boot the host from that iso?

Reply:
changed to that approach